clearwater-socket-factory/clearwater_socket_factory.cpp

```diff
@@ -18,24 +18,24 @@
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/split.hpp>
 
-const char* MGMT_SOCKET_PATH = "/tmp/clearwater_management_namespace_socket";
```

Comment:
Are you looking for me to review the C++ changes, or has @AlexHockey already reviewed these?